### PR TITLE
BETA: Register ap2.is-a.dev

### PR DESCRIPTION
**Pull request by code-hunt-in**
Added `ap2.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/ap2.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "code-hunt-in",
+        "email": "aishwary.pandore@somaiya.edu"
+    },
+    "record": {
+        "TXT": "aishwary pandore"
```
Comment (Stef-00012) [inline domains/ap2.json:7]: TXT records alone do nothing

File: domains/app.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "<code-hunt-in>",
+    "email": "<aishwary.pandore@somaiya.edu>"
+  }
+  "record": {
+    "TXT": "Aishwary Pandore"
```
Comment (Stef-00012) [inline domains/app.json:7]: TXT records alone do nothing

File: domains/app.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "<code-hunt-in>",
+    "email": "<aishwary.pandore@somaiya.edu>"
```
Comment (Stef-00012) [inline domains/app.json:4]: Invalid email (can not contain `<>`)

File: domains/app.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "<code-hunt-in>",
```
Comment (Stef-00012) [inline domains/app.json:3]: It should be your username without `<>`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**